app/routes/administrative-units/index.js

```diff
@@ -29,12 +29,7 @@ export default class AdministrativeUnitsIndexRoute extends Route {
   @dropTask({ cancelOn: 'deactivate' })
   *loadAdministrativeUnitsTask(params) {
     let query = {
-      include: [
-        'classification',
-        'province',
-        'municipality',
-        'organization-status',
-      ].join(),
+      include: ['classification', 'organization-status', 'primary-site'].join(),
```

Comment:
I would include the address as well. `primary-site.address`. Otherwise Ember data is going to fetch that when used in the template and I hate that "data" popping in effect.